src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,167 @@
+import cn from 'classnames';
+import { useState } from 'react';
+import { FilterState } from '../../types/FilterState';
+import { TempTodo } from '../../types/tempTodo';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todos: Todo[],
+  filterBy: FilterState,
+  deleteTodosFromServer: (value: number) => void
+  updateTodo: (value: Todo) => void
+  tempTodo: TempTodo | null
+  isloading: boolean
+  deletingTodoId: number | null
+};
+
+export const TodoList: React.FC<Props> = ({
+  todos,
+  filterBy,
+  deleteTodosFromServer,
+  updateTodo,
+  tempTodo,
+  isloading,
+  deletingTodoId,
+}) => {
+  const visibleTodos = todos.filter(todo => {
+    switch (filterBy) {
+      case FilterState.Active:
+        return !todo.completed;
+      case FilterState.Completed:
+        return todo.completed;
+      default:
+        return todo;
+    }
+  });
+
+  const [isEditing] = useState(false);
```

Comment:
what is it for? To me it seems that it is just a common falsy variable without the setState